sc-platform-api/src/documentation/user/getUserById.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiDocumentationParams } from '../decorators';
+import { UserResponse } from '@student-council-platform/utils';
+import { DefaultDocumentation } from '../default';
+
+export const UserDocumentationGetUserById: ApiDocumentationParams = {
+  isAuth: true,
+  ok: {
+    type: UserResponse,
+  },
+  badRequest: {
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found`,
+  },
+  unauthorized: DefaultDocumentation.UNAUTHORIZED,
```

Comment:
This response is already set as default in @ApiEndpoint decorator logic
```suggestion
  unauthorized: true,
```